odoo/tests/common.py

```diff
@@ -1636,7 +1642,13 @@ def browser_js(self, url_path, code, ready='', login=None, timeout=60, cookies=N
             timeout = timeout * 1.5
 
         self.start_browser()
-
+        if watch and self.browser._webkit_version:
+            _logger.warning('watch mode is only suitable for local testing')
+            version_hash = self.browser._webkit_version.split('@')[-1].strip((')'))
+            chrome_dev_tool_url = 'https://chrome-devtools-frontend.appspot.com/serve_file/@%s/inspector.html' % version_hash
```

Comment:
It would be better to use `devtoolsFrontendUrl` that can be found in the json endpoint